specificatie/gba-genereervariant/openapi.yaml

```diff
@@ -439,8 +441,12 @@ components:
           $ref: '#/components/schemas/Burgerservicenummer'
         opschortingBijhouding:
           $ref: '#/components/schemas/OpschortingBijhouding'
+        inOnderzoek:
+          $ref: '#/components/schemas/GbaInOnderzoek'
```

Comment:
waarom gebruik je niet gba-inonderzoek.yaml?

of andersom: waarom voeg je gba-inonderzoek toe wanneer je die niet gebruikt?

Reply:
Dit is de genereervariant. Daarin is dit geresolved. In rieidoument.yaml (waar de gba-openapi.yaml naar verwijst) gebruik ik wel de gba-inonderzoek.yaml.